pandas/core/algorithms.py

```diff
@@ -1165,7 +1163,6 @@ def compute(self, method):
 
         # slow method
         if n >= len(self.obj):
-
```

Comment:
can you revert these files as they are not actually changed

Reply:
isort fails if I don't

Reply:
ok that's odd